tasks/go_test.py

```diff
@@ -1192,6 +1193,10 @@ def get_modified_packages(ctx) -> List[GoModule]:
         if not os.path.exists(os.path.dirname(modified_file)):
             continue
 
+        # If there are no _test.go file in the folder we do not try to run tests
+        if not glob.glob(os.path.dirname(modified_file) + "/*_test.go"):
```

Comment:
```suggestion
        if not glob.glob(os.path.join(os.path.dirname(modified_file), "/*_test.go")):
```
Not sure it makes a great difference. Maybe we could use `recursive=True` as well if there could be subfolders at any moment, wdyt?

Reply:
I think it can work. It would be closer to what we have now. Updated